presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/JdbcConnectorFactory.java

```diff
@@ -35,14 +37,18 @@
         implements ConnectorFactory
 {
     private final String name;
-    private final JdbcModuleProvider moduleProvider;
+    private final JdbcModulesProvider moduleProvider;
 
     public JdbcConnectorFactory(String name, Module module)
     {
-        this(name, JdbcModuleProvider.of(module));
+        this(name, catalogName -> {
+            return ImmutableList.of(
```

Comment:
Use `import static io.airlift.configuration.ConfigurationAwareModules.combine` instead of list.